### DISCUSSION TITLE
Looking to contribute

### DISCUSSION THREAD

**Discussion by jaspermayone** [sherlock-project/sherlock#980] [General]
Hi Developers! I would love to contribute to this project and join the team. Please let me know if you are interested.

**Comment by 404oops** [comment]
What exactly is your purpose? Have you helped in issues before?

**Comment by SethFalco** [comment]
With open-source projects, you normally don't have to seek permission to contribute.
Just open issues, help with discussions, or open pull requests when you feel like it!

You can get more information on Sherlock's contribution guidelines here:
https://github.com/sherlock-project/sherlock/blob/master/CONTRIBUTING.md

If you want to contribute either code or update the site list, I recommend you learn Git first.

https://git-scm.com/

**Reply by PillowFood** [reply]
> Hi Developers! I would love to contribute to this project and join the team. Please let me know if you are interested.

If you're still trying to find a way to help, I just opened up a discussion about a feature I'd like to add.
Maybe it will get elevated to an issue that you could help code a solution to.

**Comment by Casper1012275** [comment]
Mind assisting
